include/notcurses/notcurses.h

```diff
@@ -1467,13 +1467,19 @@ API int ncplane_at_cursor_cell(struct ncplane* n, nccell* c);
 
 // Retrieve the current contents of the specified cell. The EGC is returned, or
 // NULL on error. This EGC must be free()d by the caller. The stylemask and
-// channels are written to 'stylemask' and 'channels', respectively.
+// channels are written to 'stylemask' and 'channels', respectively. The return
+// represents how the cell will be used during rendering, and thus integrates
+// any base cell where appropriate. If called upon the secondary columns of a
+// wide glyph, the EGC will be returned (i.e. this function does not distinguish
+// between the primary and secondary columns of a wide glyph).
 API char* ncplane_at_yx(const struct ncplane* n, int y, int x,
```

Comment:
`ncplane_content()` will be broken by this, since it should only have each EGC once.

Reply:
fixed, unit tested